packages/maskbook/src/extension/options-page/DashboardComponents/FixedTokenList.tsx

```diff
@@ -63,7 +65,10 @@ export function FixedTokenList(props: FixedTokenListProps) {
             tokens_ = excludeTokens.length
                 ? tokens_.filter((x) => !excludeTokens.some((y) => isSameAddress(y, x.address)))
                 : tokens_
-
+            mustIncludeTokens.forEach(
+                (token) =>
+                    !tokens_.find((token_) => isSameAddress(token.address, token_.address)) && tokens_.push(token),
```

Comment:
Better not directly change the ref object from the parameter list. You can merge `mustIncludeTokens` into `tokens` to create a new tokens list.

Reply:
You can use `immer` (in our dependency) to "mutate" the state with no pain